internal/service/alarms/internal/alertmanager/alertmanager.go

```diff
@@ -61,3 +66,103 @@ func Setup(ctx context.Context, cl client.Client) error {
 	slog.Info("Successfully configured alertmanager")
 	return nil
 }
+
+// ConvertAmToAlarmEventRecordModels get alarmEventRecords based on the alertmanager notification and AlarmDefinition
+func ConvertAmToAlarmEventRecordModels(am *api.AlertmanagerNotification, aDefinitionRecords []models.AlarmDefinition) []models.AlarmEventRecord {
+	records := make([]models.AlarmEventRecord, 0, len(am.Alerts))
+	for _, alert := range am.Alerts {
+		slog.Info("Converting Alertmanager alert", "alert name", GetAlertName(*alert.Labels))
+		record := models.AlarmEventRecord{
+			AlarmRaisedTime:   *alert.StartsAt,
+			AlarmClearedTime:  alert.EndsAt,
+			PerceivedSeverity: getPerceivedSeverity(*alert.Labels),
+			AlarmStatus:       string(*alert.Status),
+			Fingerprint:       *alert.Fingerprint,
+			ResourceID:        GetResourceID(*alert.Labels),
```

Comment:
I believe we should ignore alerts that do not have a valid resourceID (managed_cluster label), what do we get by storing them?

Reply:
Yeah basically the idea is that we collect all the alerts for now. And try to see if anything is missing later instead of  preemptively dropping rows. Given that we are now collecting them...i'm able to tell which ones are faulty ones just with quick in the DB. Cause in theory they should all have managed_cluster so nothing should be ignored